USFMToolsSharp.Renderers.Docx/DocxRenderer.cs

```diff
@@ -218,6 +259,33 @@ private void RenderFootnotes(StyleConfig styles)
                 FootnoteMarkers.Clear();
             }
         }
+        private void RenderCrossReferences(StyleConfig config)
+        {
+
+            if (CrossRefMarkers.Count > 0)
+            {
+                XWPFParagraph renderCrossRefStart = newDoc.CreateParagraph();
+                renderCrossRefStart.BorderTop = Borders.Single;
+
+                StyleConfig markerStyle = (StyleConfig)config.Clone();
+                markerStyle.fontSize = 12;
+
+                foreach (KeyValuePair<string, Marker> crossRefKVP in CrossRefMarkers)
+                {
+                    XWPFParagraph renderCrossRef = newDoc.CreateParagraph();
+                    XWPFRun crossRefMarker = renderCrossRef.CreateRun(markerStyle);
+                    crossRefMarker.SetText(crossRefKVP.Key);
+                    crossRefMarker.Subscript = VerticalAlign.SUPERSCRIPT;
+
+                    foreach (Marker marker in crossRefKVP.Value.Contents)
```

Comment:
Actually you should just call render marker on the crossRefKVP because if there is anything in the crossRefKVP as far as styling goes it isn't going to get called.

Reply:
crossRefKVP.Value.Contents only holds the parent Cross Reference marker (\x). 

I don't see why the styling wouldn't get passed to its children.

crossRefKVP = [ 1 ] : XMarker

Reply:
oh yeah. I see it. I changed it